tools/tag.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+
+import sys
+
+if len(sys.argv) < 2:
+    print("Provide version string")
+    sys.exit(1)
+
+orig = sys.argv[1]
+
+class Version:
+
+    def __init__(self, major, minor, patch, release, commit):
+        self.major = major
+        self.minor = minor
+        self.patch = patch
+        self.release = release
```

Comment:
Like you used below:

```suggestion
        self.build = build
```